packages/flutter/test/gestures/events_test.dart

```diff
@@ -248,7 +248,7 @@ void main() {
   });
 
   test('transforming events', () {
-    final Matrix4 transform = (Matrix4.identity()..scale(2.0, 2.0, 1.0)) * Matrix4.translationValues(10.0, 20.0, 0.0);
+    final Matrix4 transform = (Matrix4.identity()..scale(2.0, 2.0, 1.0)) * Matrix4.translationValues(10.0, 20.0, 0.0) as Matrix4;
```

Comment:
nit: I would slightly prefer to just say 

```dart
final Matrix4 transform = (Matrix4.identity()..scale(2.0, 2.0, 1.0)).multiplied(Matrix4.translationValues(10.0, 20.0, 0.0));
```

Here and elsewhere.

Reply:
Done